packages/jest-runner/src/run_test.js

```diff
@@ -113,26 +122,20 @@ export default function runTest(
       result.console = testConsole.getBuffer();
       result.skipped = testCount === result.numPendingTests;
       result.displayName = config.displayName;
-      return result;
-    })
-    .then(
-      result =>
-        Promise.resolve().then(() => {
-          environment.dispose();
-          if (globalConfig.logHeapUsage) {
-            if (global.gc) {
-              global.gc();
-            }
-            result.memoryUsage = process.memoryUsage().heapUsed;
-          }
-
-          // Delay the resolution to allow log messages to be output.
-          return new Promise(resolve => setImmediate(() => resolve(result)));
-        }),
-      err =>
-        Promise.resolve().then(() => {
-          environment.dispose();
-          throw err;
-        }),
-    );
+      if (environment.teardown) await environment.teardown();
```

Comment:
this needs curly braces, wondering why CI didn't catch that :O

Reply:
https://github.com/prettier/eslint-config-prettier/blob/0bfb266df652850857dc0340af41e6bc2a43434f/README.md#curly